showmehow/showmehow.py

```diff
@@ -108,21 +106,14 @@ def show_response_scrolled(value):
     is to enable newlines to be printed correctly without extraneous whitespace
     on either side.
     """
-    print_lines_slowly(in_blue("\n".join(itertools.chain.from_iterable([
-        textwrap.wrap(v)
-        for v in value.splitlines()
-    ]))))
-
-def show_raw_response(value):
-    """Print text as-is."""
-    print(value)
+    print_lines_slowly(in_blue(value))
 
 
 _RESPONSE_ACTIONS = {
-    "raw": show_raw_response,
-    "scrolled": show_response_scrolled,
+    "raw": show_response,
+    "scrolled": print_lines_slowly,
```

Comment:
`show_response_scrolled` was effectively just going to call `print_lines_slowly` directly if it was left in, since we are not using text wrapping now. I just figured I'd cut out the indirection.